fenix/app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -104,18 +103,20 @@ private fun HomeSectionHeaderContent(
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
-                    .semantics {
-                        contentDescription = description
-                    },
-                style = TextStyle(
-                    color = showAllTextColor,
-                    fontSize = 14.sp,
-                ),
-                onClick = { onShowAllClick() },
-            )
+            TextButton(onClick = { onShowAllClick() }) {
+                Text(
+                    text = stringResource(id = R.string.recent_tabs_show_all),
+                    style = TextStyle(
+                        color = showAllTextColor,
+                        fontSize = 14.sp,
+                    ),
+                    modifier = Modifier
+                        .padding(start = 16.dp)
+                        .semantics {
+                            contentDescription = description
+                        },
```

Comment:
Even with using named parameters we try to respect the declaration order for method parameters.
For `Text` the `modifier` should be above `style`, the same as in the current code.